test/test_artifacts/v1/altair.test.Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG COSMOS_IMAGE
+FROM $COSMOS_IMAGE
+
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+
+CMD ["python", "-c", "import altair"]
```

Comment:
Is this the best we can do? Can we go deeper and test more of the package's functionality or is it like we don't have appropriate test cases today that we can rely on?